### ISSUE TITLE
How to enable autofix ?

### ISSUE THREAD

**Issue by mrleblanc101** [nuxt-community/stylelint-module#24]
Hi,
I was using Stylelint Webpack Plugin by adding this to my config:
```
config.plugins.push(
    new StylelintPlugin({
        files: ['src/**/*.vue', 'src/**/*.scss'],
        fix: true,
    }),
);
```
but I just say create-nuxt-app was updated to add stylelint option, but I don't see a way to enable autofix.

**Comment by mrleblanc101** [comment]
Just found the answer, I'll let this open as the doc could be improved to add information about this
If anybody has the same issue:
```
buildModules: ['@nuxtjs/stylelint-module'],
stylelint: {
    fix: true,
},
```
Same as eslint config option

**Comment by ricardogobbosouza** [comment]
Hi @mrleblanc101 
You can pass all options from stylelint https://github.com/nuxt-community/stylelint-module#options

**Comment by mrleblanc101** [comment]
I've opened pull request #25 to document this as it's the only missing option.

**Comment by ricardogobbosouza** [comment]
Merged #25 

Thanks @mrleblanc101
